src/app/core/user-module/user.service.ts

```diff
@@ -154,14 +119,11 @@ export class UserService {
     const route = USER_ROUTES.GET_SAME_ORGANIZATION(organization);
```

Comment:
This needs to be updated

Reply:
I just deleted the route, it's not used anywhere.